src/styles/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    padding-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    text-decoration: none;
+    color: #060b35;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14.06px;
+    height: 60px;
+    position: relative;
+    transition: color $transition-duration;
+
+    &:hover {
+      color: $active-nav;
+    }
+  }
+
+  &__item:last-child {
+    padding-right: 0;
+  }
+}
+
+.logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  box-sizing: border-box;
+}
+
+.is-active {
+  color: $active-nav;
```

Comment:
Ensure that the variable `$active-nav` is defined in your SCSS files. If it's not defined, this will cause a compilation error.